pyulog/core.py

```diff
@@ -74,7 +74,7 @@ def get_field_size(type_str):
     _unpack_uint64 = struct.Struct('<Q').unpack
 
 
-    def __init__(self, file_name, message_name_filter_list=None):
+    def __init__(self, file, message_name_filter_list=None):
         """
         Initialize the object & load the file from disk.
 
```

Comment:
Can you please add a parameter description for `file`, now that it's not trivial anymore?